### ISSUE TITLE
Nav in VoiceOver landmarks does not include secondary nav menu 

### ISSUE THREAD

**Issue by erinysong** [cisagov/getgov#4019]
When testing, we found when the user selects the nav (Primary navigation navigation) in the VO landmarks menu, the landmark only includes items in the primary navigation. This follows the behavior of the VO landmarks on the USWDS header. However, not labeling our enterprise mode USWDS secondary nav (the top nav with user profile and sign out options) may be misleading for users who use VoiceOver.

We should determine if this is something we should change from USWDS's VoiceOver defaults and implement any changes, either by consulting USWDS or implementing our own solution.

> In the Landmarks menu in VO, there is no landmark for secondary nav, so I'm unable to get to the profile/log off. Maybe that menu (in enterprise mode) should be wrapped in a nav? In legacy, I do hear that has the primary nav, but in enterprise it is not.
For clarification, in legacy, the profile nav is the primary nav and shows up in landmark. In enterprise, the primary nav is the one where the domains and orgs show up, and the profile section does not show up as a navigation menu.

> I'm looking at our current production code and [USWDS's base code for their secondary and primary navs in their extended header](https://federalist-3b6ba08e-0df4-44c9-ac73-6fc193b0e19c.sites.pages.cloud.gov/preview/uswds/uswds/develop/iframe.html?id=components-header--extended&args=&viewMode=story) and when I use VO there it looks like their secondary navs are excluded from showing up as a navigation menu there as well.
It's interesting that happens since both USWDS's primary and secondary nav divs are wrapped within a single nav tag :/
I'm not as familiar on best practices on VO landmarks but is there a good alternative behavior for this? I can also flag this to USWDS to see if this is intended behavior on their end

_Originally posted by @erinysong in https://github.com/cisagov/manage.get.gov/pull/3968#issuecomment-3129937660_

**Comment by erinysong** [comment]
cc: @kimallen who also noticed and brought up the VO experience on our nav

**Comment by lizpearl** [comment]
with @CocoByte: Could be a 2 or 3 depending on approach
